src/services/webdav/backend.rs

```diff
@@ -408,6 +445,28 @@ impl WebdavBackend {
         self.client.send_async(req).await
     }
 
+    async fn webdav_put(
+        &self,
+        path: &str,
+        size: Option<u64>,
+        content_type: Option<&str>,
+        body: AsyncBody,
+    ) -> Result<Response<IncomingAsyncBody>> {
+        self.webdav_custom_request(BodyRequestType::Put, path, size, content_type, body)
```

Comment:
Please expand them directly.

Reply:
Do you mean calling `self.webdav_custom_request(BodyRequestType::Put, path, size, content_type, body)` inline?

Reply:
> Do you mean calling `self.webdav_custom_request(BodyRequestType::Put, path, size, content_type, body)` inline?

I mean remove `self.webdav_custom_request` and write the logic here directly.

Reply:
updated. Thanks!